blogposts/2021/show-us-your-calendar.md

```diff
@@ -4,151 +4,163 @@ description: 'Engineers are leading the way toward rethinking the 40 hour work w
 
-Vanesa was struggling to juggle a full time job, a new baby, and her portion of the household work. Weekends were no longer a space for recovery; they became stressful days for housework and childcare: “I was exhausted after the weekend.”
+Vanesa was struggling to juggle a full-time job, a new baby, and her portion of the household work. Weekends were no longer a space for recovery; they became stressful days for housework and childcare: "I was exhausted after the weekend."
 
-Together, Vanesa and her then engineering manager decided to try a four day work week–it was an immediate success. Communicating the change to her team only required marking the time in Google Calendar and the occasional reminder that she was unavailable for a Friday meeting. The schedule served her during her time as an engineer and she’s maintained it since becoming a community advocate.
+Together, Vanesa and her then engineering manager decided to try a four-day work week–it was an immediate success. Communicating the change to her team only required marking the time in Google Calendar and the occasional reminder that she was unavailable for a Friday meeting. The schedule served her during her time as an engineer and she's maintained it since becoming a community advocate.
 
-Now, she has the time she needs: “On the weekend I get to enjoy my children. It’s not like I’m trying to push them away as I deal with household chores.” The success of her new schedule has lasted through to her second child. “I can’t imagine going back,” she says, “especially with the second kid.”
+Now, she has the time she needs: "On the weekend I get to enjoy my children. It's not like I'm trying to push them away as I deal with household chores." The success of her new schedule has lasted through to her second child. "I can't imagine going back," she says, "especially with the second kid."
 
-Vanesa recommends the four-day work week to everyone, but especially to primary caretakers: “If you are struggling with work/life balance, and you need that extra time to practice self care and to deal with all the messes in your life, then it should really be considered.”
+Vanesa recommends the four-day work week to everyone, but especially to primary caretakers: "If you are struggling with work/life balance, and you need that extra time to practice self care and to deal with all the messes in your life, then it should really be considered."
 
-Vanesa is careful to note, however, that anyone considering changing their schedule needs to do a cost-benefit analysis. Vanesa works 32 hours across 4 days instead of 40, meaning she has to consider the consequences of working part-time.
+Vanesa is careful to note, however, that anyone considering changing their schedule needs to do a cost-benefit analysis. Vanesa works 32 hours across four days instead of 40 across 5, meaning she has to consider the consequences of working part time.
```

Comment:
https://about.sourcegraph.com/handbook/communication/content_guidelines/style_and_mechanics#dashes-and-hyphens